src/Handlebars.Net.Helpers/Helpers/StringHelpers.cs

```diff
@@ -142,6 +142,48 @@ public string Lowercase(string value)
             return value.ToLower();
         }
 
+        [HandlebarsWriter(WriterType.String)]
+        public string PadLeft(string value, object width, string padChar)
```

Comment:
`object width` --> `int totalWidth`

Reply:
I used "object" because the documentation called for being able to parse strings as well as int's.

Reply:
I changed the wiki to only allow an integer. Please change object to int.

Reply:
Will do.